src/routes/(admin)/dashboard/+page.server.ts

```
@@ -30,5 +56,65 @@ export const load: PageServerLoad = async () => {
 };
+export const actions: Actions = {
+	applyFilter: async ({ request, cookies }) => {
+		const token = getSessionTokenCookie();
+
+		const form = await superValidate(request, zod(basedateSchema));
+		if (!form.valid) {
+			return fail(400, {
+				form
+			});
+		}
+		const dateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)